packages/cli/src/main/kotlin/elide/tool/io/RuntimeWorkdirManager.kt

```diff
@@ -48,16 +51,16 @@ import elide.tool.io.WorkdirManager.WorkdirHandle
 
   internal companion object {
     private const val nativesDir = "native"
-    private const val cachesDir = "caches"
+    // private const val cachesDir = "caches"
     private const val flightRecorderDir = "blackbox"
-    private const val nixTempPath = "/tmp/elide-runtime"
-    private const val runtimeDirPrefix = "elide-runtime-"
-    private const val elideHomeDirectory = ".elide"
-    private const val elideConfigDirectory = "elide"
-    private const val configDirectory = ".config"
-
-    private val linuxCachesPath = "~/elide/caches/v${Elide.version()}"
-    private val darwinCachesPath = "/Library/caches/elide/v${Elide.version()}"
+    // private const val nixTempPath = "/tmp/elide-runtime"
+    // private const val runtimeDirPrefix = "elide-runtime-"
+    // private const val elideHomeDirectory = ".elide"
+    //private const val elideConfigDirectory = "elide"
+    //private const val configDirectory = ".config"
+
+    // private val linuxCachesPath = "~/elide/caches/v${Elide.version()}"
+    // private val darwinCachesPath = "/Library/caches/elide/v${Elide.version()}"
```

Comment:
`const val` must be listed literally, because it's "constant." on the other hand, `const` can be dropped and you can just use `@JvmStatic val`, which is a "static value," that is computed at program _startup time_, instead of build time.